lib/class-wp-rest-block-navigation-areas-controller.php

```diff
@@ -37,7 +37,8 @@ public function register_routes() {
 					'permission_callback' => array( $this, 'get_items_permissions_check' ),
 					'args'                => $this->get_collection_params(),
 				),
-				'schema' => array( $this, 'get_public_item_schema' ),
+				'schema'      => array( $this, 'get_public_item_schema' ),
+				'allow_batch' => array( 'v1' => true ),
```

Comment:
We are now following this pattern. 

https://github.com/WordPress/wordpress-develop/commit/ecf1d6a158c70dd7f10cc42c4b26ecb0fb22bb59

So this should be 
```
'allow_batch' => $this->allow_batch,
```

Reply:
Good to know.
This task is already merged, but I will be using `'allow_batch' => $this->allow_batch` from now on.